R/tm_variable_browser.R

```diff
@@ -534,6 +534,7 @@ var_missings_info <- function(x) {
 #' @return character variable containing the HTML code of the \code{sparkline} HTML widget
 #' @keywords internal
 #'
+#' @export
```

Comment:
Should this be exported?

Maybe we should have a call about this and finish the discussion: https://github.com/insightsengineering/coredev-tasks/issues/510

Reply:
Good idea :+1: 
Having read your most recent comment, I lean towards removing this tag.

Reply:
Me too, between Kartik comment and that conscious review I think we can keep the current behaviour.

If you remove this export I can approve the PR without waiting for that discussion to finish. Waiting 1 day also wouldn't block anything.

Up to you.